src/simplay-web/src/EventSerialized.ts

```diff
@@ -0,0 +1,8 @@
+import { EventAction } from './EventAction';
+
+export interface EventSerialized {
+  forId: string;
+  timestamp: number;
+  action: EventAction;
+  args: Record<string, unknown>;
```

Comment:
Is there a way to get rid of unknown?

Reply:
sadly, I don't think so :(

Linting suggested me this change, I had `any` before